docs/howto/quickstart.md

```diff
@@ -0,0 +1,281 @@
+# Quick start application deployment using ASO
+
+In this guide we will:
+ - Create a Service Principal
+ - Create an Azure Container Registry
+ - Create an AKS cluster with the ACR attached
+ - Deploy ASO
+ - Deploy an Azure Votes app that requires Azure SQL Server using ASO
+
+## Set up the Kubernetes environment and deploy ASO
+
+ ### Requirements
+- az cli ([install instructinos](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest))
+- kubectl cli ([install instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/))
+- helm cli ([install instructions](https://helm.sh/docs/intro/install/))
+- jq [install instructions](https://stedolan.github.io/jq/download/)
+
+### Prepare credentials and variables
+
+Set these variables now as they will help with copy/paste of commands later.
+
+>Only the first 3 environment variables need to be changed before pasting.
+
+```
+VOTES_APP_INSTANCE_NAME="<alias>" # only alpha numeric
+TENANT_ID=""
+SUBSCRIPTION_ID=""
```

Comment:
hmm, yes, this looks better to me

Reply:
although....not sure if the windows folks will get this to work in powershell...

Reply:
guess since we are already using jq it doesn't matter too much

Reply:
We could use `--query` and `--output tsv` instead of jq, similar to the other examples in the file:

```
TENANT_ID=$(az account show --query homeTenantId -o tsv)
SUBSCRIPTION_ID=$(az account show --query id -o tsv)
```